frontend/packages/dev-console/src/components/pipelines/pipeline-builder/task-sidebar/TaskSidebarParamArray.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { FieldArray } from 'formik';
+import { Flex, FlexItem, FlexModifiers, FormGroup, TextInputTypes } from '@patternfly/react-core';
+import { MinusCircleIcon } from '@patternfly/react-icons';
+import { global_disabled_color_200 as disabledColor } from '@patternfly/react-tokens';
+import { InputField } from '@console/shared';
+import MultiColumnFieldFooter from '@console/shared/src/components/formik-fields/multi-column-field/MultiColumnFieldFooter';
```

Comment:
I like the idea of having this component to be shared as a formik field so that we can reuse it in other use cases as well. That way there's no need to import this `Footer` component like this as well.

Reply:
Okay, will look to give it a home under shared.

Reply:
I'll push a comment saying to refactor after Debsmita merges her PR. She already did this no sense in my complicating my PR with a duplicate implementation.